apps/payroll/README.md

```diff
@@ -0,0 +1,9 @@
+# Payroll
+
+The purpose of app to implement a Payroll system in multiple currencies.
+
+### TODO / Possible improvements
+
+- Allow employees to change payment period, and of course adapt time restriction for `payday` to it. (Thanks to @monicazng !)
+- Would it be better to define allowed tokens only globally instead of per employee? Not sure, pros and cons, needs more discussion about use cases. Roughly: per employee if makes sense to allow different tokens for different employees; globally would make way easier to add (or remove) allowed tokens.
+- Finish documentation and move it to [Aragon's wiki](https://wiki.aragon.one)
```

Comment:
@izqui Yes, we should look into it, we might need to write a [plugin](http://www.mkdocs.org/user-guide/plugins/) for it if one doesn't exist, since we can't pull external content into the wiki as easily. Will look into how we could do this.